routing/graph.go

```diff
@@ -86,3 +92,38 @@ func (g *CachedGraph) fetchNodeFeatures(nodePub route.Vertex) (
 
 	return g.graph.FetchNodeFeatures(nodePub)
 }
+
+// FetchAmountPairCapacity determines the maximal capacity between two nodes.
+//
+// NOTE: Part of the routingGraph interface.
+func (g *CachedGraph) FetchAmountPairCapacity(nodeFrom, nodeTo route.Vertex,
+	amount lnwire.MilliSatoshi) (btcutil.Amount, error) {
+
+	// For the local node we assume no information on the channel capacity.
```

Comment:
Why is this necessary? And is it the right point to do this check? From a distance it looks like a pair capacity could still be returned for local channels.

Reply:
Right, I removed it from the function, for this we already have the local probability calculation and it is a layer violation.